src/main/java/mate/academy/spring/dao/UserDaoImpl.java

```diff
@@ -0,0 +1,51 @@
+package mate.academy.spring.dao;
+
+import java.util.List;
+import mate.academy.spring.model.User;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class UserDaoImpl implements UserDao {
+
+    private final SessionFactory sessionFactory;
+
+    @Autowired
```

Comment:
We need to use @Autowired  annotation above constructors everywhere or nowhere to make our code more consistent. Remove it everywhere or add it everywhere